force_bdss/core/data_value.py

```diff
@@ -0,0 +1,34 @@
+from traits.api import HasStrictTraits, Any, String, Enum
+
+
+class DataValue(HasStrictTraits):
```

Comment:
Should we specify where does the DataValue come from ? Like a `from` trait:
```
from = Enum('MCO', 'MODEL', 'DATASOURCE', 'KPI')
```
Or will this be part of the slots ?